src/theme.js

```diff
@@ -0,0 +1,47 @@
+import { createTheme } from '@mui/material/styles';
+
+const theme = createTheme({
+  typography: {
+    fontFamily: [
```

Comment:
I am curious as to why there's so many font families in here? Can we not just select one, like "Roboto"

Reply:
They're used as backups incase the prior ones cannot be loaded, but yeah we can cut down to one or two